source/frontend/src/components/RegionManagement/DiscoverableRegions/DiscoverableAccountsTable.js

```diff
@@ -46,14 +48,135 @@ const columns = [
     id: 'regions',
     header: 'Regions',
     cell: (e) => e.regionCount,
-    width: 200,
-    minWidth: 200,
+    width: 125,
+    minWidth: 125,
   },
+  {
+      id: 'iamRoleStatus',
+      header: 'Role Status',
+      cell: (e) => {
+          if(e.isIamRoleDeployed == null) return <StatusIndicator type='info'>Unknown</StatusIndicator>;
```

Comment:
Do you think warning might be more appropriate than info?

Reply:
I prefer info because nothing has actually gone wrong yet. We will know on the next run of the discovery process whether it has and then we can step up to warning if required.